### ISSUE TITLE
PDO Port definition

### ISSUE THREAD

**Issue by stebogit** [SchwarzerIT/laravel-mysql-to-seeder#2]
Hello @AndreSchwarzer thanks for your tool!

I noticed you can't defined the port for you DB connection, so I just added that.
I don't rally have time to create a PR and stuff, but it's just a very quick fix if you want to add it to your code:
```php
public function __construct(
        $hostname,
        $databasename,
        $username = null,
        $password = null,
        $port = 3306, // <-- added
        $driver_options = array(parent::MYSQL_ATTR_INIT_COMMAND => 'SET NAMES utf8'),
        $prefix = null,
        $suffix = null
    ) {
        date_default_timezone_set('Europe/Berlin');

        $this->dsn = 'mysql:dbname=' . $databasename . ';host=' . $hostname 
                . ';port=' . $port;  // <-- added
        $this->databasename = $databasename;
// ....
```

**Comment by AndreSchwarzer** [comment]
Thanks @stebogit for your input, I totally forgot about GitHub since I moved, all, except this, projects to GitLab.

I've added the port at the end of the argument list for compatibility reasons.

In the next week or two there will be a fresh new version of this package, allowing to use it in Laravel (> 4.2) as package and artisan command or as standalone version. And a few more cool features.

This will be versioned v1.* because it'll cause braking changes.
